backend/controllers/Auth/RegisterController.js

```diff
@@ -11,6 +11,8 @@ class RegisterController {
 
         const registeredUser = await this.userService.findById(user.id);
 
+        delete registeredUser.password;
```

Comment:
by default `password` shouldnt be fetched by `findById`, it should be excluded in `defaultScope`

Reply:
I was looking for best place to do it in couchdb but i think this is the best way to just remove key from object in `LoginController` and `RegisterController`. In couchdb we have no models like in sequelize to exclude in defaultScope

Reply:
ok, so maybe change `findById` and remove password there because password should be fetched only during login

Reply:
I did something similar, added new method `getUserByIdWithExcludedPassword` special for this 2 cases, maybe will be ok :P Are you sure that password should be fetched during login ? I thought we just send credentials to api and check there is credentials are valid

Reply:
I meant you have to fetch password and compare it with sending password in request, of course you can not return this passwrod to user, after comparing you have to fetch user again without password